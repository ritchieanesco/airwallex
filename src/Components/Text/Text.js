import React from "react";
import Typography from "@material-ui/core/Typography";
import { node, string } from "prop-types";

const Text = props => (
  <Typography variant={props.variant} gutterBottom={true}>
    {props.children}
  </Typography>
);

Text.propTypes = {
  variant: string,
  children: node
};

export default Text;
