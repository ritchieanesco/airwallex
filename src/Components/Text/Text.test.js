import React from "react";
import renderer from "react-test-renderer";
import Text from "./Text";

describe("Footer", () => {
  it("should render correctly", () => {
    const tree = renderer
      .create(<Text variant="display3">A better way to enjoy every day</Text>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
