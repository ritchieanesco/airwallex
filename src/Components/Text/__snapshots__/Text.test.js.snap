// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render correctly 1`] = `
<h1
  className="MuiTypography-root-1 MuiTypography-display3-3 MuiTypography-gutterBottom-18"
>
  A better way to enjoy every day
</h1>
`;
